components/layout/__tests__/navigation.test.tsx

```diff
@@ -52,7 +52,7 @@ describe("Navigation", () => {
             >
               <a
                 class="inline-flex flex-col items-center justify-center text-center h-12 px-2 rounded-md text-gray-600 hover:text-blue-600"
-                href="https://wa.me/6281257579812"
+                href="https://bit.ly/hotlinewarga"
```

Comment:
This test will be failing once the editor update the link through Netlify CMS. Please supply a hardcoded variable instead for the test. If it's not supplied through props, you can mock the value using `jest.mock` utility.

Reply:
The link is mocked in ab2d255 and the links are asserted in 00f585e